src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -844,8 +873,25 @@ li {
     font-weight: normal;
   }
 
+  .v-input--checkbox .v-input__control .v-input__slot .v-label.theme--light {
+    padding-top: 2px;
+  }
+
+  .v-input.sp-confirm-label {
+    margin-bottom: 30px !important;
```

Comment:
1. This input element already declares class `mb-6` to set the bottom margin. Just update this.
2. This style now overrides the element's helper class. Delete this.

Ref: https://vuetifyjs.com/en/styles/spacing/

Rationale: keep this simple/clean. It's far easier to update layouts by using the helper classes right in the template -- you can tell right away how an element should be padded, etc, instead of having to look up custom class names.

Reply:
Also, as a general rule in our projects (*), we specify top whitespace instead of bottom whitespace. In other words, an element knows how much white space should be above it and not care above white space below it (which is the following element's concern).

_Sometimes_ we specify bottom whitespace for the last element in a block (so that the block has the right whitespace all around). This is the exception.

_Sometimes_ we specify negative bottom margin to make the element "shorter" than default so it fits in its visual bounding box (ie, no whitespace below it). This depends on the element in question and its layout on the page.

On this page, I'd prefer to see top whitespace used instead of bottom whitespace or a mix. I can show you on a call why I think this makes sense.

(*) Other projects may use exclusively bottom whitespace. That's OK as long as it's consistent, though IMO it makes conditional elements more difficult to lay out.

Reply:
I forgot to remove this style after I made the change. My bad.

Reply:
Next time we can look at changing bottom whitespace to top whitespace. This is fine for now.